x-pack/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/integration_tests/status_action.test.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { AppContextTestRender } from '../../../../../common/mock/endpoint';
+import { createAppRootMockRenderer } from '../../../../../common/mock/endpoint';
+import {
+  ConsoleManagerTestComponent,
+  getConsoleManagerMockRenderResultQueriesAndActions,
+} from '../../../console/components/console_manager/mocks';
+import React from 'react';
+import { getEndpointConsoleCommands } from '../../lib/console_commands_definition';
+import { enterConsoleCommand } from '../../../console/mocks';
+import { getEndpointAuthzInitialState } from '../../../../../../common/endpoint/service/authz';
+import type { EndpointCapabilities } from '../../../../../../common/endpoint/service/response_actions/constants';
+import { ENDPOINT_CAPABILITIES } from '../../../../../../common/endpoint/service/response_actions/constants';
+import { useGetEndpointPendingActionsSummary } from '../../../../hooks/response_actions/use_get_endpoint_pending_actions_summary';
+import { useGetEndpointDetails } from '../../../../hooks';
+import { EndpointActionGenerator } from '../../../../../../common/endpoint/data_generators/endpoint_action_generator';
+import { EndpointMetadataGenerator } from '../../../../../../common/endpoint/data_generators/endpoint_metadata_generator';
+
+jest.mock('../../../../../common/experimental_features_service');
```

Comment:
FYI: The `mockedContext` return by `reateAppRootMockRenderer()` already mocks all of the FF and also provides you with a method `setExperimentalFlag()` that allows you to change them for your test.

Reply:
removed it 097ac5b410e1a531b86a826e74fb884f3dfe2929